### PR TITLE
kie-issues#562: replace copyrights headers/comments in kogito-apps

### LINKED ISSUE #562: replace copyrights headers/comments in `kogito-apps`

### PR DESCRIPTION
**Pull request by jomarko**
Closes: https://github.com/kiegroup/kie-issues/issues/562

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/apache/incubator-kie-kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- for <b>pull request checks</b>  
  Please add comment: <b>Jenkins retest this</b>

- for a <b>specific pull request check</b>  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] tests</b>

- for <b>quarkus branch checks</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins run quarkus-branch</b>

- for a <b>quarkus branch specific check</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-branch</b>

- for <b>quarkus main checks</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins run quarkus-main</b>

- for a <b>specific quarkus main check</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-main</b>

- for <b>quarkus lts checks</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins run quarkus-lts</b>

- for a <b>specific quarkus lts check</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-lts</b>
 
- for <b>native checks</b>  
  Run native checks  
  Please add comment: <b>Jenkins run native</b>

- for a <b>specific native check</b>  
  Run native checks 
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native</b>

- for <b>native lts checks</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins run native-lts</b>

- for a <b>specific native lts check</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native-lts</b>
 
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

<details>
<summary>
Quarkus-3 PR check is failing ... what to do ?
</summary>
The Quarkus 3 check is applying patches from the `.ci/environments/quarkus-3/patches`.

The first patch, called `0001_before_sh.patch`, is generated from Openrewrite `.ci/environments/quarkus-3/quarkus3.yml` recipe. The patch is created to speed up the check. But it may be that some changes in the PR broke this patch.  
No panic, there is an easy way to regenerate it. You just need to comment on the PR:
```
jenkins rewrite quarkus-3
```
and it should, after some minutes (~20/30min) apply a commit on the PR with the patch regenerated.

Other patches were generated manually. If any of it fails, you will need to manually update it... and push your changes.
</details>

### CODE REVIEW

**Review by yesamer — APPROVED** [review]

**Comment by kie-ci3** [comment]
**PR job** `#5` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/display/redirect

**Test results:**
- PASSED: 1626
- FAILED: 9

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.index.infinispan/ProcessDataIndexInfinispanKafkaIT/testProcessInstanceEvents/">org.kie.kogito.index.infinispan.ProcessDataIndexInfinispanKafkaIT.testProcessInstanceEvents</a></summary>
1 expectation failed.<br/>JSON path data.UserTaskInstances[0].attachments.size() doesn't match.<br/>Expected: is <1><br/>  Actual: <0><br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.index.oracle/ProcessDataIndexOracleKafkaIT/testProcessInstanceEvents/">org.kie.kogito.index.oracle.ProcessDataIndexOracleKafkaIT.testProcessInstanceEvents</a></summary>
java.util.concurrent.CompletionException: java.lang.RuntimeException: Unable to start Quarkus test resource class org.kie.kogito.index.test.quarkus.kafka.DataIndexOracleQuarkusKafkaTestResource
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.it.trusty/QuarkusTrustyExplainabilityEnd2EndIT/doTest/">org.kie.kogito.it.trusty.QuarkusTrustyExplainabilityEnd2EndIT.doTest</a></summary>
Container startup failed
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.it.trusty/SpringBootTrustyExplainabilityEnd2EndIT/doTest/">org.kie.kogito.it.trusty.SpringBootTrustyExplainabilityEnd2EndIT.doTest</a></summary>
Container startup failed
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.index.service/KeycloakInfinispanIndexingServiceIT/testUnauthorizedUserAccess/">org.kie.kogito.index.service.KeycloakInfinispanIndexingServiceIT.testUnauthorizedUserAccess</a></summary>
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors<br/> [error]: Build step io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor#startKafkaDevService threw an exception: java.lang.RuntimeException: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor.startKafkaDevService(DevServicesKafkaProcessor.java:105)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/> at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/> at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/> at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)<br/> at io.quarkus.builder.BuildContext.run(BuildContext.java:281)<br/> at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)<br/> at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)<br/> at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)<br/> at java.base/java.lang.Thread.run(Thread.java:829)<br/> at org.jboss.threads.JBossThread.run(JBossThread.java:501)<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)<br/> at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)<br/> at io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor.lambda$startKafka$5(DevServicesKafkaProcessor.java:236)<br/> at java.base/java.util.Optional.orElseGet(Optional.java:369)<br/> at io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor.startKafka(DevServicesKafkaProcessor.java:282)<br/> at io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor.startKafkaDevService(DevServicesKafkaProcessor.java:95)<br/> ... 11 more<br/>Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)<br/> ... 16 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:542)<br/> at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)<br/> ... 17 more<br/>Caused by: java.lang.IllegalStateException: Container exited with code 1<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:514)<br/> ... 19 more<br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.explainability.rest/KeycloakExplainabilityServiceIT/shouldReturnUnauthorized/">org.kie.kogito.explainability.rest.KeycloakExplainabilityServiceIT.shouldReturnUnauthorized</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.security/KeycloakSecurityCommonsServiceIT/meTest/">org.kie.kogito.security.KeycloakSecurityCommonsServiceIT.meTest</a></summary>
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors<br/> [error]: Build step io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor#startKeycloakContainer threw an exception: java.lang.RuntimeException: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:247)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/> at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/> at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/> at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)<br/> at io.quarkus.builder.BuildContext.run(BuildContext.java:281)<br/> at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)<br/> at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)<br/> at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)<br/> at java.base/java.lang.Thread.run(Thread.java:829)<br/> at org.jboss.threads.JBossThread.run(JBossThread.java:501)<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)<br/> at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.lambda$startContainer$4(KeycloakDevServicesProcessor.java:372)<br/> at java.base/java.util.Optional.orElseGet(Optional.java:369)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startContainer(KeycloakDevServicesProcessor.java:397)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:195)<br/> ... 11 more<br/>Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)<br/> ... 16 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:542)<br/> at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)<br/> ... 17 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Keycloak.*started.*'<br/> at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:47)<br/> at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:52)<br/> at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:953)<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:485)<br/> ... 19 more<br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.trusty.service.common/KeycloakTrustyServiceIT/___/">org.kie.kogito.trusty.service.common.KeycloakTrustyServiceIT.(?)</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/5/testReport/org.kie.kogito.trusty.service.common.api/DecisionsApiV1IT/testGetExecutionById/">org.kie.kogito.trusty.service.common.api.DecisionsApiV1IT.testGetExecutionById</a></summary>
java.lang.reflect.InvocationTargetException
</details>

**Comment by kie-ci3** [comment]
**PR job** `#6` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/6/display/redirect

**Test results:**
- PASSED: 368
- FAILED: 4

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/6/testReport/org.kie.kogito.examples.quarkus/GrafanaDockerComposeIT/___/">org.kie.kogito.examples.quarkus.GrafanaDockerComposeIT.(?)</a></summary>
Local Docker Compose not found. Is docker-compose on the PATH?
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/6/testReport/org.kie.kogito.quarkus.outbox/OutboxIT/___/">org.kie.kogito.quarkus.outbox.OutboxIT.(?)</a></summary>
Local Docker Compose not found. Is docker-compose on the PATH?
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/6/testReport/org.kie.kogito.examples.springboot/GrafanaDockerComposeIT/___/">org.kie.kogito.examples.springboot.GrafanaDockerComposeIT.(?)</a></summary>
Local Docker Compose not found. Is docker-compose on the PATH?
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/6/testReport/org.kie.kogito.springboot.outbox/OutboxIT/___/">org.kie.kogito.springboot.outbox.OutboxIT.(?)</a></summary>
Local Docker Compose not found. Is docker-compose on the PATH?
</details>

**Comment by kie-ci3** [comment]
**PR job** `#1` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/1/display/redirect

**Test results:**
- PASSED: 1633
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/1/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/1/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/1/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

File: NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Apache KIE
```
Comment (yesamer) [inline NOTICE.txt]: @jomarko NOTICE file shouldn't have the .txt extension

**Comment by kie-ci3** [comment]
**PR job** `#9` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/9/display/redirect

**Test results:**
- PASSED: 1633
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/9/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/9/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/9/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Comment by kie-ci3** [comment]
**PR job** `#10` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/10/display/redirect

**Test results:**
- PASSED: 2776
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/10/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/10/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/10/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Comment by jstastny-cz** [comment]
The 3 test failures are known instabilities targeted in another PR by Enrique, good to go ahead with merge once reviewers approve the changes. @paulovmr do you want to review or should we try to find someone else?

**Comment by kie-ci3** [comment]
**PR job** `#12` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1883 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/12/display/redirect

**Test results:**
- PASSED: 2750
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/12/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/12/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1883/12/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Review by jstastny-cz — APPROVED** [review]
LGTM,Failed tests are known issues

**Pull request merged**